editor/plugins/canvas_item_editor_plugin.cpp

```diff
@@ -4111,6 +4122,8 @@ CanvasItemEditor::CanvasItemEditor(EditorNode *p_editor) {
 
 	bone_last_frame = 0;
 
+	_update_scrollbars();
```

Comment:
I told you this was not good in my patch ;)
```
ERROR: get_edited_scene_root: Index current_edited_scene=-1 out of size (edited_scene.size()=0)
   At: editor/editor_data.cpp:614.
ERROR: get_edited_scene_root: Index current_edited_scene=-1 out of size (edited_scene.size()=0)
   At: editor/editor_data.cpp:614.
```

Reply:
The problem is elsewhere though. It should be able to update the scrollbars even if no scene is edited.

Reply:
`_update_scrollbars()` calls `editor->get_edited_scene()` twice, likely what triggers the error when there is no edited scene.

Reply:
```
int EditorData::get_edited_scene() const {
        return current_edited_scene;
}
```
`current_edited_scene` defaults to `-1` when there is no edited scene, so the checks should likely be `if (get_edited_scene() != -1)`

Reply:
Wait that's the wrong `get_edited_scene()`, the one used is:
```
Node *get_edited_scene() { return editor_data.get_edited_scene_root(); }
```
in `EditorNode`.

Reply:
Hum that's weird that get_edited_scene_root displays an error when there's no scene

Reply:
Try `if (editor_data->get_edited_scene_count())`.

Reply:
Actually `if (editor->get_editor_data().get_edited_scene_count())`

Reply:
I just moved the _update_scrollbar to NOTIFICATION_ENTER_TREE. It works fine, so I would not bother more. ^^